src/DOMTestTrait.php

```diff
@@ -75,12 +84,73 @@ public static function assertSelectRegExp($selector, $pattern, $count, $actual,
      * @param mixed                 $actual
      * @param string                $message
      * @param boolean               $isHtml
+     * @return void
+     * @throws Exception
+     *
      * @since Method available since Release 1.0.0
      *
-     * @throws \Exception
      */
     public static function assertSelectEquals($selector, $content, $count, $actual, $message = '', $isHtml = true)
     {
-        DOMTestCase::assertSelectEquals($selector, $content, $count, $actual, $message, $isHtml);
+        $crawler = new Crawler;
+
+        if ($actual instanceof \DOMDocument) {
+            $crawler->addDocument($actual);
+        } elseif ($isHtml) {
+            $crawler->addHtmlContent($actual);
+        } else {
+            $crawler->addXmlContent($actual);
+        }
+
+        $crawler = $crawler->filter($selector);
+
+        if (is_string($content)) {
+            $crawler = $crawler->reduce(static function (Crawler $node) use ($content) {
+                $text = $node->text(null, true);
+
+                if ($content === '') {
+                    return $text === '';
+                }
+
+                if (preg_match('/^regexp\s*:\s*(.*)/i', $content, $matches)) {
+                    return (bool)preg_match($matches[1], $text);
+                }
+
+                return strstr($text, $content) !== false;
+            });
+        }
+
+        $found = count($crawler);
+
+        if (is_numeric($count)) {
+            Assert::assertEquals($count, $found, $message);
+        } elseif (is_bool($count)) {
+            if ($count) {
+                Assert::assertGreaterThan(0, $found, $message);
+            } else {
+                Assert::assertEquals(0, $found, $message);
+            }
+        } elseif (is_array($count) &&
+            (isset($count['>']) || isset($count['<']) ||
+                isset($count['>=']) || isset($count['<=']))) {
+
+            if (isset($count['>'])) {
+                Assert::assertGreaterThan($count['>'], $found, $message);
```

Comment:
Using `assertGreaterThan` variants gives the user a better error message than `assertTrue/assertFalse`